packages/react-vapor/src/ReactVaporReducers.ts

```diff
@@ -63,6 +64,7 @@ export const ReactVaporReducers: ReducersMapObject = {
     dropdowns: dropdownsReducer,
     dropdownSearch: dropdownsSearchReducer,
     facets: facetsReducer,
+    filepickers: FilepickerReducers,

```

Comment:
If we follow the existed pattern, should we name it as `filepickers: filepickersReducer,`